chapters/lexicalstructure.tex

```diff
@@ -247,6 +247,24 @@ \subsection{Strings}\label{strings}
 \end{lstlisting}
 \end{nonnormative}
 
+\subsection{Units of Literal Constants}\label{units-literal-constants}
+
+Literal \lstinline!Real! and \lstinline!Integer! constants are implicitly assumed to have \lstinline!unit = "1"! if they appear in expressions containing variables, parameters, or constants that have a non-empty \lstinline!unit! attribute string.
+
+The inputs of transcendental built-in functions such as \lstinline!sin()!, \lstinline!cos()!, \lstinline!exp()!, etc., should have a dimensionless unit (e.g., \lstinline!"1"| or \lstinline!"rad"!) if they involve expressions containing variables, parameters, or constants that have a non-empty unit attribute string.
+Their outputs are implicitly assumed to have unit \lstinline!"1"! if they are called within expressions containing variables, parameters, or constants that have a non-empty \lstinline!unit! attribute string.
```

Comment:
```suggestion
Their outputs are also implicitly assumed to have a dimensionless unit if they are called within expressions containing variables, parameters, or constants that have a non-empty \lstinline!unit! attribute string.
```
This makes it symmetric for input and output, without spending more text on it - so handling asin as inverse of sin.

Reply:
Sounds good. It is up to the tool implementors to figure out how to handle this.